uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/queue/AsyncDownloadIntegrationTest.java

```diff
@@ -131,11 +136,21 @@ void sendAndProcessMessageSuccessfullyAfterRetry() throws IOException {
         // Producer
         verify(this.messageService, never()).alreadyProcessed(jobId);
         await().until(jobCreatedInRedis(jobId));
-        await().until(jobErrored(jobId));
+        //        await().until(jobErrored(jobId));
+        await().atLeast(Duration.ofMillis(100))
+                .atMost(Duration.ofSeconds(15))
+                .with()
+                .pollInterval(Duration.ofMillis(100))
+                .until(jobErrored(jobId));
         // verify  redis
         verifyRedisEntry(query, jobId, List.of(JobStatus.ERROR), 1, true);
```

Comment:
Please try to keep only atMost and until and test. If test fails you can keep all config.